torch/_inductor/scheduler.py

```diff
@@ -581,6 +583,16 @@ def get_estimated_runtime(self) -> float:
             # default to no reordering based on runtime
             return 0
 
+        # Collective kernels
```

Comment:
you had to move this higher up since it was getting bypassed previously?

Reply:
Yes. Previously this part always failed and returned 0 because for collective kernels, the dtype obtained above is always `None`.

```python
        try:
            gpu_memory_bandwidth = get_gpu_dram_gbps()
            gpu_flops = get_device_tflops(dtype) * 10**12
        except Exception:
            return 0
```